paasta_tools/vitess_tools.py

```diff
@@ -52,61 +52,9 @@ def __init__(
             branch_dict=branch_dict,
         )
 
-    def get_service_name_smartstack(self) -> str:
```

Comment:
This particular method is fine https://github.com/Yelp/paasta/blob/master/paasta_tools/long_running_service_tools.py#L170, are all the ones with default NotImplemented exceptions needed to be implemented? I don't see that in case of cassandracluster_tools or kafkacluster_tools either

Reply:
I would probably implement the ones that cassandra/kafka have implemented - unless y'all won't be calling these (and/or none of the PaaSTA internals try to call these implicitly)